app/portainer/settings/edge-compute/SettingsFDO/SettingsFDO.tsx

```diff
@@ -162,8 +156,32 @@ export function SettingsFDO({ settings, onSubmit }: Props) {
               </Form>
             )}
           </Formik>
+
+          {edgeComputeFeaturesEnabled && isFDOEnabled && (
+            <>
+              <br />
+              <FormSectionTitle>Device Profiles</FormSectionTitle>
+              <TextTip color="blue">
+                Add, Edit and Manage the list of device profiles available
+                during FDO device setup
+              </TextTip>
+              <FDOProfilesDatatableContainer />
+            </>
+          )}
         </WidgetBody>
       </Widget>
     </div>
   );
+
+  async function onChangedEnabled(
+    e: boolean,
+    setFieldValue: (
+      field: string,
+      value: unknown,
+      shouldValidate?: boolean
+    ) => void
+  ) {
+    setIsFDOEnabled(e.valueOf());
```

Comment:
why do we need the `.valueOf()`? why not just `e`?

Reply:
probably copy/paste from an example I found, removed it